cmd/containerboot/main.go

```diff
@@ -993,6 +1140,16 @@ func (s *settings) validate() error {
 	return nil
 }
 
+func resolveDNS(ctx context.Context, name string) ([]net.IP, error) {
+	ips, err := net.LookupIP(name)
```

Comment:
Consider using `net.DefaultResolver.LookupIP(ctx, name)` to respect any deadlines on ctx.